### DISCUSSION TITLE
Issue in https://www.graphql-java.com/tutorials/getting-started-with-spring-boot

### DISCUSSION THREAD

**Discussion by himanshuntwk** [graphql-java/graphql-java#2678] [General]
Hi Team,
I observe one minor issue in this tutorial (https://www.graphql-java.com/tutorials/getting-started-with-spring-boot), not sure where to raise so I am using discussions.
The issue is getPageCountDataFetcher method should fetch "pageCount" instead of "totalPages".
This >>  return book.get("totalPages");
should be >> return book.get("pageCount");

I am not sure about the exact repository, otherwise I will raise an issue and corresponding PR for this.
![Screenshot 2022-01-16 at 8 51 57 PM](https://user-images.githubusercontent.com/13043702/149666225-8d215d86-b1a0-4e2f-99e6-41283a125db9.png)

**Comment by jord1e** [comment]
> I am not sure about the exact repository, otherwise I will raise an issue and corresponding PR for this.

https://github.com/graphql-java/graphql-java-page/blob/master/src/pages/tutorials/getting-started-with-spring-boot.md#default-datafetchers

you should probably start using https://github.com/Netflix/dgs-framework (stable) or https://github.com/spring-projects/spring-graphql (experimental)

**Reply by himanshuntwk** [reply]
Thanks a lot, for guiding me in the right direction.
On further analysis, I observe it's not an issue, instead intentionally done to demonstrate key mismatch scenario in DataFetchers.
I have created a PR to make it more friendly, to help people like me :)
